infra/roles/stores/tasks/main.yml

```diff
@@ -36,3 +36,24 @@
 
 - name: Copy testfile
   copy: src=index.php dest=/var/www/stores/index.php mode=0664
+
+- name: Set up authorized keys
+  authorized_key:
+    user: ubuntu
+    state: present
+    key: '{{ item }}'
+  with_file:
+    - ssh_key.pub
```

Comment:
This name needs to match the name of the file, which currently is set to `stores_devops.pub`